projectile.el

```diff
@@ -419,7 +419,7 @@ it for functions working with buffers."
   :type '(repeat string))
 
 (defcustom projectile-globally-ignored-buffers nil
-  "A list of buffer-names ignored by projectile.
+  "A list of regular expressions for buffer-names ignored by projectile.
```

Comment:
I think this change is a bit misleading given the name of the defcustom. Might make more sense to just add one more defcustom for regular expressions instead.

Reply:
```
(defcustom projectile-globally-ignored-modes
  '("erc-mode"
    "help-mode"
    "completion-list-mode"
    "Buffer-menu-mode"
    "gnus-.*-mode"
    "occur-mode")
  "A list of regular expressions for major modes ignored by projectile.

If a buffer is using a given major mode, projectile will ignore
it for functions working with buffers."
  :group 'projectile
  :type '(repeat string))
```

Reply:
IMHO it might seem a bit redundant to have two defcustoms `projectile-globally-ignored-buffers` and `projectile-globally-ignored-buffers-regexp`, but I can do that if you think it's better solution.